Resources/Prototypes/_CM14/Entities/Structures/Machines/colmartech_vendors.yml

```diff
@@ -237,16 +237,16 @@
     #  amount: 3
     - name: Extended Ammunition
       entries: []
-    #- id: CMMagazineExtendedM39
-    #  amount: 1
-    #- id: CMMagazineExtendedM41A
-    #  amount: 1
+    - id: CMMagazineExtendedCaselessM39
+      amount: 1
+    - id: CMMagazineExtendedRifleM41A
+      amount: 1
     - name: Special Ammunition
       entries: []
     #- id: CMDrumSmartgunM56
     #  amount: 1
-    #- id: CMHeavySpeedLoaderM44
-    #  amount: 2
+    - id: CMSpeedLoaderM44
+      amount: 2
```

Comment:
This is where the error comes from. These magazines and speed loader are being ignored because the YAML is invalid. Both entries are blank with `entries: []` and CMMagazineExtendedCaselessM39, CMMagazineExtendedRifleM41A and CMSpeedLoaderM44 are wrongly indented.
You can see this in-game:
![image](https://github.com/CM-14/CM-14/assets/10968691/bdb02703-100e-4d0d-bb9f-92a204b7dc7e)

It should be:
```yaml
    - name: Extended Ammunition
      entries:
      - id: CMMagazineExtendedCaselessM39
        amount: 1
      - id: CMMagazineExtendedRifleM41A
        amount: 1
    - name: Special Ammunition
      entries:
    #- id: CMDrumSmartgunM56
    #  amount: 1
      - id: CMSpeedLoaderM44
        amount: 2
```

Reply:
Fixed in [`2273af5` (#935)](https://github.com/CM-14/CM-14/pull/935/commits/2273af52180f403127bc1d50de11da6d77f9759a)

Reply:
Ah.. that's what I get for not just reformatting. Yeah some of the format with the original commenting was messed up